cnd/migrations/2020-05-06-105000_created-swap-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE swaps
```

Comment:
Should be single according to the `drop` migration.

Reply:
Oh, thanks.  The `DROP TABLE foo` statements are all wrong.